scr/main/java/bittercode/model/User.java

```diff
@@ -0,0 +1,88 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class User implements Serializable {
+
+    private String emailId;
+    private String password;
+    private String firstName;
+    private String lastName;
+    private Long phone;
+    private Char address;
+    private int<UserRole> roles;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/2#issuecomment-2419037001>#be98fd</a><span> - 10/17/2024, 09:42 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The change of the 'roles' field from List&lt;UserRole&gt; to int&lt;UserRole&gt; introduces an invalid Java type. Could you please review this change and consider using a valid Java type for storing user roles?
</div>
<div id="fix">
<b>Fix:</b> Change the data type of the 'roles' field back to List&lt;UserRole&gt; to allow for proper storage of user roles and ensure compilation.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -14,1 +14,1 @@
 -    private int<UserRole> roles;
 +    private List<UserRole> roles;
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged